contactPage/contactPage.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Joseph:Home</title>
+</head>
+<body>
+    <nav>
+        <div class="navContainer">
+            <div class="logoName">
+                JaHolt
```

Comment:
If you have the opening tag, content, and closing tag all on one line it will take up less space in your html file and will still display the same in a browser.

Reply:
On that note, the logo does not display in the same font on this page as it does on the other page.

Reply:
Resolved